README.md

```diff
@@ -36,7 +37,7 @@ In command line:
 
 ```bash
 # npm module
-npm install sass-true
+npm install sass-true node-sass
```

Comment:
I'm not sure if this requires more explanation or not? Most users will probably already have `node-sass` installed?

Reply:
I think it's worth explaining for the people that don't have it.